ui/page/components/components.go

```diff
@@ -664,3 +664,20 @@ func TranslateErr(err error) string {
 
 	return err.Error()
 }
+
+// CoinImageBySymbol returns image widget for supported asset coins.
+func CoinImageBySymbol(icons *load.Icons, coinName string) *decredmaterial.Image {
+	m := map[string]*decredmaterial.Image{
+		"btc": icons.BTC,
+		"dcr": icons.DCR,
+		"bch": icons.BCH,
+		"ltc": icons.LTC,
+	}
+	coin, ok := m[strings.ToLower(coinName)]
+
+	if !ok {
+		return nil
+	}
+
+	return coin
```

Comment:
Wonder if a switch might be better here rather than a map of several icons where only one is needed, or maybe even none if an unexpected coin name is passed:

```suggestion
	switch strings.ToLower(coinName) {
	case "btc":
		return icons.BTC
	case "dcr":
		return icons.DCR
	case "bch":
		return icons.BCH
	case "ltc":
		return icons.LTC
	}
	return nil
```